app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view("insert");
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $post= new Post;
+        $post->title= $request->get('title');
+        $post->body= $request->get('body');
+        $post->save();
+        return redirect("show");
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Post  $post
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Post $post)
+    {
+        $posts = Post::all();
+        return view('show', ['posts'=>$posts]);
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  \App\Models\Post  $post
+     * @return \Illuminate\Http\Response
+     */
+    public function edit(Post $post, $id)
+    {
+        $posts = Post::find($id);
+        return view('edit',['posts'=>$posts]);
+
+
```

Comment:
thừa dòng trắng ở dưới nhiều nè xóa bớt đi nha.
Sau dấu phẩy thêm cho m 1 dấu cách nha. Và tương tự như trên cách ra giữa return và khối lệnh còn lại
```suggestion
        $posts = Post::find($id);

        return view('edit', compact(['posts']));
```